### PR TITLE
[4.0] Front - Make the category articles list table responsive

### PR DESCRIPTION
**Pull request by shim-sao**
Pull Request for Issue # .

### Summary of Changes
Make the category articles list table responsive on the frontend if the table width is larger than the client width like mobiles.

### Testing Instructions
Display a category with a table of articles list and multiples columns (dates, hits...) to have a large table.

### Actual result BEFORE applying this Pull Request
If the table is larger than the client, the table exceeds in width and the entire template is scrollable.

### Expected result AFTER applying this Pull Request
The table overflow if hidden and only the table is scrollable

### CODE REVIEW

File: build/media_source/system/js/joomla-toolbar-button.w-c.es6.js [hunk 1]
```diff
@@ -101,9 +101,6 @@ window.customElements.define('joomla-toolbar-button', class extends HTMLElement
       return false;
     }
 
-    // Avoid multiple call to the task like accidental double-click
-    this.disabled = true;
```
Comment (dgrammatiko) [inline build/media_source/system/js/joomla-toolbar-button.w-c.es6.js]: Why?

**Comment by BertaOctech** [comment]
I have tested this item :white_check_mark: successfully on 62fd856c97dd9a06554ba66f06c4c3faf26bb9e5<br /><br />After following the Testing Instructions, the result is as expected.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34378">issues.joomla.org/tracker/joomla-cms/34378</a>.</sub>

**Comment by maikol-ortigueira** [comment]
I have tested this item :white_check_mark: successfully on 62fd856c97dd9a06554ba66f06c4c3faf26bb9e5<br /><br />I have tested this item successfully.
Joomla versión: 4.0.0-rc1
PHP 7.4.13
Thank's!!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34378">issues.joomla.org/tracker/joomla-cms/34378</a>.</sub>

**Comment by bembelimen** [comment]
If you want to make the tables responsive you should use: https://getbootstrap.com/docs/5.0/content/tables/#responsive-tables

**Comment by dgrammatiko** [comment]
@bembelimen yeah, strongly recommended and also badly broken 🤣 
Screenshot 2021-06-04 at 17 52 06

**Comment by bembelimen** [comment]
It's "broken by design", because they use the "wrong" breakpoints (for the demo). So in general it should work with ```table-responsive``` or the correct break point suffix.

**Comment by dgrammatiko** [comment]
I will definitely embrace the term `broken by design`™©®. Describes perfectly my code 😂

**Pull request closed without merging**